app/src/main/java/com/devshish/internship/presentation/ui/utils/SnackbarExt.kt

```diff
@@ -0,0 +1,17 @@
+package com.devshish.internship.presentation.ui.utils
+
+import android.view.View
+import androidx.annotation.IdRes
+import androidx.annotation.StringRes
+import com.devshish.internship.R
+import com.google.android.material.snackbar.Snackbar
+
+fun Snackbar.caseSmthWentWrong(
+    @IdRes anchorViewRes: Int = R.id.bottomNavView,
+    @StringRes actionMessageRes: Int = R.string.snackbar_retry,
+    action: ((View) -> Unit)?
+) {
+    setAnchorView(anchorViewRes)
+    setAction(actionMessageRes, action)
+    show()
```

Comment:
This extension is useless and wrong on all levels
How about use something like this:

![telegram-cloud-photo-size-2-5294276834705389703-y](https://user-images.githubusercontent.com/23121846/145007486-f9bb8b97-a91d-4440-b786-d960c753f492.jpg)